e2e/testsuite/grpc_query.go

```diff
@@ -6,29 +6,9 @@ import (
 	intertxtypes "github.com/cosmos/interchain-accounts/x/inter-tx/types"
 	"github.com/strangelove-ventures/ibctest/ibc"
 
-	clienttypes "github.com/cosmos/ibc-go/v5/modules/core/02-client/types"
 	channeltypes "github.com/cosmos/ibc-go/v5/modules/core/04-channel/types"
-	ibcexported "github.com/cosmos/ibc-go/v5/modules/core/exported"
 )
 
-// QueryClientState queries the client state on the given chain for the provided clientID.
-func (s *E2ETestSuite) QueryClientState(ctx context.Context, chain ibc.Chain, clientID string) (ibcexported.ClientState, error) {
-	queryClient := s.GetChainGRCPClients(chain).ClientQueryClient
-	res, err := queryClient.ClientState(ctx, &clienttypes.QueryClientStateRequest{
-		ClientId: clientID,
-	})
-	if err != nil {
-		return nil, err
-	}
-
-	clientState, err := clienttypes.UnpackClientState(res.ClientState)
-	if err != nil {
-		return nil, err
-	}
-
-	return clientState, nil
-}
-
```

Comment:
this function was going to be used in one of @colin-axner 's coming tests I believe. I don't think we should remove this.

Reply:
Wow... I actually have no idea how this got removed. I never touched it 🤣 
I'll put it back!